docs/how-to-guides/monitoring-with-prometheus-operator.md

```diff
@@ -0,0 +1,139 @@
+# Setting up monitoring on Lokomotive
+
+## Contents
+
+* [Introduction](#introduction)
+* [Prerequisites](#prerequisites)
+* [Deploy Prometheus Operator](#deploy-prometheus-operator)
+  * [Configure Prometheus Operator](#configure-prometheus-operator)
+  * [Install Prometheus Operator](#install-prometheus-operator)
+* [Accessing Prometheus Operator sub-components](#accessing-prometheus-operator-sub-components)
+  * [Accessing Prometheus](#accessing-prometheus)
+  * [Accessing Alertmanager](#accessing-alertmanager)
+  * [Accessing Grafana](#accessing-grafana)
+    * [Using port forward](#using-port-forward)
+    * [Using Ingress](#using-ingress)
+
+## Introduction
+
+This guide provides installation steps of monitoring stack using prometheus-operator component and ways to access the sub-components like Prometheus, Alertmanager and Grafana.
```

Comment:
We could link here to prometheus-operator component documentation.

Reply:
Yes it does in the `### Configure Prometheus Operator` section.

Reply:
Also added it in `Additional resources`.